metadata_utilities.py

```diff
@@ -73,7 +73,7 @@ def _serialize_xml(write, elem, encoding, qnames, namespaces):
 # END MONKEYPATCH CDATA
 
 
-def generate_iso_metadata(project_definition=None):
+def generate_iso_metadata(supplemental_information=None):
```

Comment:
What you suggest makes sense. Anyway, I think in this context supplemental_information is a good name, because I am building the metadata to be uploaded to geonode, where I am going to use the supplemental_information field. We could change the name inside QGIS, but I think it is intuitive to use the same name we are using on the platform, so the correspondence between the two things is more immediate. For now, I would prefer to keep the name as it is, but I am open to discuss further about it.